internal/testing/kind-node-labels.yaml

```diff
@@ -21,6 +21,13 @@ nodes:
       nodeRegistration:
         kubeletExtraArgs:
           node-labels: "node.kubernetes.io/role=worker"
+- role: worker
```

Comment:
Why duplicating this patch? it is already defined right above

Reply:
In this test, I'm creating three workers and parking two of them (each with different labels).